modules/core/02-client/keeper/events.go

```diff
@@ -96,6 +96,21 @@ func emitUpdateClientProposalEvent(ctx sdk.Context, clientID, clientType string)
 	})
 }
 
+// emitRecoverClientEvent emits a recover client event
+func emitRecoverClientEvent(ctx sdk.Context, clientID, clientType string) {
```

Comment:
people's thoughts, do we want separate events? raised this with Charly too https://github.com/cosmos/ibc-go/pull/4436#discussion_r1308880005